Gruntfile.js

```diff
@@ -392,19 +409,22 @@ module.exports = function(grunt) {
                   echo 'ERROR: Links found with target="_blank" but no rel="noopener noreferrer" set.  Please add required rel attribute.')`,
       },
       'setup-admin': {
+        stderr: true,
         cmd:
-          'curl -X PUT http://localhost:5984/_node/${COUCH_NODE_NAME}/_config/admins/admin -d \'"pass"\'' +
-          ' && curl -X POST http://admin:pass@localhost:5984/_users ' +
+          `curl -X PUT ${couchConfig.withPathNoAuth(couchConfig.dbName)}` +
+          ` && curl -X PUT ${couchConfig.withPathNoAuth('_users')}` +
+          ` && curl -X PUT ${couchConfig.withPathNoAuth('_node/${COUCH_NODE_NAME}/_config/admins/admin')} -d '"${couchConfig.password}"'` +
+          ` && curl -X POST ${couchConfig.withPath('_users')} ` +
           ' -H "Content-Type: application/json" ' +
-          ' -d \'{"_id": "org.couchdb.user:admin", "name": "admin", "password":"pass", "type":"user", "roles":[]}\' ' +
-          ' && curl -X PUT --data \'"true"\' http://admin:pass@localhost:5984/_node/${COUCH_NODE_NAME}/_config/chttpd/require_valid_user' +
-          ' && curl -X PUT --data \'"4294967296"\' http://admin:pass@localhost:5984/_node/${COUCH_NODE_NAME}/_config/httpd/max_http_request_size',
+          ` -d '{"_id": "org.couchdb.user:${couchConfig.username}", "name": "${couchConfig.username}", "password":"${couchConfig.password}", "type":"user", "roles":[]}' ` +
+          ` && curl -X PUT --data '"true"' ${couchConfig.withPath('_node/${COUCH_NODE_NAME}/_config/chttpd/require_valid_user')}` +
```

Comment:
is the interaction between javascript's and bash's string interpolation going to be problematic?

Reply:
I agree that having two sources of interpolation in the same string is weird... especially since they use the same syntax. I've made all interpolations in js.

Reply:
Though, it is a maintability problem -- it is working as-is.  There is a clear order - Javascript first and then Bash. Anything output from js with the ${} anchors is left to bash.

Reply:
I'm surprised javascript wouldn't just error, e.g.:
```
> `${nothing}`
ReferenceError: nothing is not defined
```

Reply:
But here it is `${'nothing'}` where the undefined value is inside a single quote string